config/sample-idex.cfg

```diff
@@ -44,13 +50,30 @@ gcode:
 
 # Activate the primary extruder
 [gcode_macro T0]
+variable_offset_applied: 0
 gcode:
-    PARK_{printer.toolhead.extruder}
+    {% set svv = printer.save_variables.variables %}
+    {% if "x" in printer.toolhead.homed_axes %} # this check ensures compatibility with Cura
+        PARK_{printer.toolhead.extruder}
+    {% endif %}
+    {% set fan_speed = printer["gcode_macro M106"].swap_speed %}
+      {% if fan_speed != -1 %}
+        SET_FAN_SPEED FAN=fan_extruder SPEED={fan_speed}
```

Comment:
I wonder what is supposed to happen to the fan speed of the other extruder? Would it make sense to really swap their speeds, so that the fan of the parked extruder get the fan speed of the previously parked extruder?

Reply:
So, using CURA, if you tell it you have separate tools it will set the fan speed of each manually and the variable_swap_speed will stay -1 (Because P is set) and let CURA control everything.

If using PrusaSlicer, it won't emit the P{n} so it expects the active tool's part fan to be controlled.

Currently having the part fan of the parked extruder run will cool somewhere on the bed that isn't where printing is happening. In theory this unnecessary draft could cause part warping so I set it to turn off entirely to minimize risks.
This is based on the assumption that the heatsink fan is adequate to not cause the tool to melt.